rotkehlchen/db/optimismtx.py

```diff
@@ -31,11 +31,12 @@ def add_evm_transactions(
             relevant_address,
         )
 
-        tx_tuples = [(tx.l1_fee, tx.tx_hash) for tx in evm_transactions]
+        serialized_chainid = evm_transactions[0].chain_id.serialize_for_db()
```

Comment:
Hmm I just noticed something. This is a good change, but not future-proof. If at any point  we change `add_evm_transactions()` to accept transactions of multiple chain ids this would break.

Add it with `serialize_for_db()` and back in the loop. We already do similar in `super().add_evm_transactions(`

We could optimize by taking out of the loop both here and there for now. **But** we would also need to either programatically (assert) or by types check all passed `evm_transactions` to this function have the same `chain_id`.

As they say, premature optimization is the root of all evil, so just keep it simple for now